Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine
+
+RUN apk add --no-cache make git bash
+
+WORKDIR /keepalived-exporter
+
+COPY . .
+
+RUN make build
```

Comment:
Please use multistage build to separate the building stage and the final image.

Reply:
Do you mind having this with #58?